guardian/prosemirror-typerighter
# Update typedoc and rebuild documentation

## What does this change?

Updates our typedoc plugins, which were broken for unknown reasons, and regenerates our documentation.

There are known issues with typedoc's [library mode](https://github.com/TypeStrong/typedoc/pull/1184) that mean the generated type information is incomplete, particularly references to interfaces, but this is still an improvement on what was there before.

I've also added/updated some of the comments in the documentation to better describe the public APIs.

## How to test

The readme in ./docs should reflect the readme in ./build.

The type information referenced in the readme should exist and be ... moderately useful, although it can definitely be improved!
